plugins/utils/emmet.nix

```diff
@@ -12,23 +12,42 @@ with helpers.vim-plugin;
     originalName = "emmet-vim";
     defaultPackage = pkgs.vimPlugins.emmet-vim;
     globalPrefix = "user_emmet_";
+
+    # TODO introduced 2024-03-01: remove 2024-05-01
     deprecateExtraConfig = true;
+    optionsRenamedToSettings = [
+      "mode"
+    ];
+    imports = [
+      (
+        mkRenamedOptionModule
+        ["plugins" "emmet" "leader"]
+        ["plugins" "emmet" "settings" "leader"]
```

Comment:
shouldn't this be `leader_key` in the second list?

Reply:
Yes indeed... good catch